app/src/main/res/layout/activity_main.xml

```diff
@@ -7,17 +7,58 @@
         android:layout_height="match_parent"
         tools:context=".MainActivity">
 
-    <TextView
-            android:id="@+id/main_text"
-            android:textSize="42sp"
-            android:layout_margin="5sp"
-            android:textAlignment="center"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:text="Hello World!"
-            app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintLeft_toLeftOf="parent"
-            app:layout_constraintRight_toRightOf="parent"
-            app:layout_constraintTop_toTopOf="parent"/>
+<!--    <TextView-->
+<!--        android:id="@+id/main_text"-->
+<!--        android:layout_width="wrap_content"-->
+<!--        android:layout_height="wrap_content"-->
+<!--        android:layout_margin="5sp"-->
+<!--        android:text="Hello World!"-->
+<!--        android:textAlignment="center"-->
+<!--        android:textSize="42sp"-->
+<!--        app:layout_constraintBottom_toBottomOf="parent"-->
+<!--        app:layout_constraintHorizontal_bias="0.475"-->
+<!--        app:layout_constraintLeft_toLeftOf="parent"-->
+<!--        app:layout_constraintRight_toRightOf="parent"-->
+<!--        app:layout_constraintTop_toTopOf="parent"-->
+<!--        app:layout_constraintVertical_bias="0.574" />-->
+
+    <Spinner
+        android:id="@+id/station_names_spinner_departure"
+        android:layout_width="wrap_content"
+        android:layout_height="0dp"
+        android:layout_marginStart="88dp"
+        android:layout_marginTop="148dp"
+        android:layout_marginEnd="323dp"
+        android:entries="@array/stations_array"
+        android:onClick="onSubmitButtonTapped"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <Spinner
+        android:id="@+id/station_names_spinner_destination"
```

Comment:
kinda rogue that you guys have chosen departure and destination :joy: perhaps stick with either origin/destination or departure/arrival?